pkg/config/config.go

```diff
@@ -19,9 +22,11 @@ func fetchEnv(varString string, fallbackString string) string {
 }
 
 func LoadConfig() *Config {
+	timeSecond, _ := strconv.Atoi(fetchEnv("TIMEOUT_CONTEXT", "5"))
```

Comment:
I like the use of defaults here. Nevertheless, ideally the default should be a bit more than 5 seconds 😆 . Let's set this to 30 minutes.
```suggestion
	timeSecond, _ := strconv.Atoi(fetchEnv("TIMEOUT_CONTEXT", "1800"))
```